pywinauto/windows/uia_element_info.py

```diff
@@ -95,6 +95,8 @@ class UIAElementInfo(ElementInfo):
         "top_level_only": ("depth", {True: 1, False: None}),
     }
 
+    use_raw_view_walker = False
```

Comment:
Maybe it's worth writing docstring for this important global attribute. It should appear here after merge: https://pywinauto.readthedocs.io/en/atspi/code/pywinauto.windows.uia_element_info.html

Reply:
Ok, I wrote this: 
`"""Enable/disable RawViewWalker-based implementation (can find more elements in some cases, but slow)"""`